### PR TITLE
Subscription change email

### PR DESCRIPTION
**Pull request by nickpell**
implements http://manage.dimagi.com/default.asp?158407

@benrudolph @biyeun

### CODE REVIEW

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -752,6 +755,24 @@ def apply_upgrades_and_downgrades(self, downgraded_privileges=None,
             if not upgrade_handler.get_response():
                 raise SubscriptionChangeError("The upgrade was not successful.")
 
+        if (
+            not (old_subscription and old_subscription.is_trial)
+            or not (new_subscription and new_subscription.plan_version.plan.edition == SoftwarePlanEdition.COMMUNITY)
```
Comment (dimagimon) [inline corehq/apps/accounting/models.py]: E501 line too long (117 > 115 characters)

**Comment by benrudolph** [comment]
i like the negative amounts ;)

**Comment by nickpell** [comment]
I'm like oh shit this pull request doesn't have negative numbers what did I
do wrong ...

**Comment by benrudolph** [comment]
my bad clicked the link in skype thinking it was the one i was just looking at

**Comment by nickpell** [comment]
haha no worries

**Comment by nickpell** [comment]
thanks!

**Pull request merged**